pkg/agent/openflow/pipeline_test.go

```diff
@@ -21,9 +21,19 @@ import (
 	"go.uber.org/mock/gomock"
 
 	"antrea.io/antrea/pkg/agent/config"
+	nodeiptest "antrea.io/antrea/pkg/agent/nodeip/testing"
 	oftest "antrea.io/antrea/pkg/agent/openflow/testing"
+	binding "antrea.io/antrea/pkg/ovs/openflow"
+	ovsoftest "antrea.io/antrea/pkg/ovs/openflow/testing"
+	"antrea.io/antrea/third_party/proxy"
+	"antrea.io/libOpenflow/openflow15"
+	"github.com/stretchr/testify/require"
 )
 
+// Referring to page 43 in https://opennetworking.org/wp-content/uploads/2014/10/openflow-switch-v1.5.1.pdf,
+// an OpenFlow message size can reach up to 64kB.
+const openflowMessageMaxSize uint32 = 64000
```

Comment:
64KB should be `64 * 1024 * 1024`?

Reply:
I made a fundamental mistake. It should be `64*1024`, we use byte here.